content/en/boilerplates/gateway-api-remove-crds.md

```diff
@@ -0,0 +1,7 @@
+---
+---
+Remove the Kubernetes Gateway API CRDs:
```

Comment:
Is there an "if installed" option, and would it make sense to remove both the standard and the experimental in the same instructions?

(See previous comment about {{< k8s_gateway_api_version >}} too)

Reply:
It might, but I would rather do that in a followup - for this doc (waypoints) all we care about is the `standard` option.

Reply:
I think ultimately we will need an `gateway-api-{install/remove}-{experimental/standard}-crds set of boilerplates.